reana_client/cli/workflow.py

```diff
@@ -635,8 +656,16 @@ def workflow_stop(ctx, workflow, force_stop, access_token):
 @add_access_token_options
 @click.pass_context
 def workflow_run(ctx, file, filenames, name, skip_validation,
-                 access_token, parameters, options):
-    """Create, upload and start wrapper command."""
+                 access_token, parameters, options):  # noqa: D301
+    """Create, upload and start a new analysis workflow.
+
+    The `run` command permits to create a new analysis workflow, upload its
+    input files and start it in one command.
+
+    Examples: \n
+    \t $ reana-client run \n
+    \t $ reana-client run -f reana.yaml -n workflow.6 code/helloworld.py
```

Comment:
Is `code/helloworld.py` wanted there?  A more common example may be:

reana-client run -n myanalysis-test-small -p myparam=mysmallvalue
reana-client run -n myanalysis-test-big -p myparam=mybigvalue